src/instructlab/model/evaluate.py

```diff
@@ -865,6 +889,36 @@ def evaluate(
             display_branch_eval_summary(
                 Benchmark.MMLU_BRANCH, improvements, regressions, no_changes
             )
+
+        elif benchmark == Benchmark.UNITXT:
+            # Third Party
+            from instructlab.eval.unitxt import UnitxtEvaluator
+
+            evaluator = UnitxtEvaluator(
+                model,
+                unitxt_recipe,
+            )
+
+            server = None
+            try:
+                server, api_base, _ = launch_server(
+                    ctx,
+                    model,
+                    model,
+                    None,
+                    gpus,
+                    backend,
+                    enable_serving_output,
+                )
+                overall_scores, individual_scores = evaluator.run(api_base)
+            finally:
+                if server is not None:
+                    server.shutdown()
+
+            print("# KNOWLEDGE EVALUATION REPORT")
+            print("\n## MODEL (SCORES)\n")
+            print(overall_scores)
```

Comment:
This output format doesn't seem consistent with the rest of evaluate

Reply:
Please explain. Would you prefer I only printed a single score instead of the returned dictionary and used the display_model() format?

Reply:
I don't know the format well enough to say what's desired.  Is the example in the test typical?